ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -57,6 +55,7 @@ export const AnacondaWizard = ({ dispatch, isBootIso, osRelease, storageData, lo
     const [isFormValid, setIsFormValid] = useState(false);
     const [stepNotification, setStepNotification] = useState();
     const [isInProgress, setIsInProgress] = useState(false);
+    const [isInstallationProgress, setInstallationProgress] = useState(false);
```

Comment:
There is no need (it's weird) for this to be state variable. It should just be constant.

const inInstallationProgressStep = path[0] === "installation-progress";